### ISSUE TITLE
IGE confirmation in Intake inbox

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#1023]
* As an Intake analyst
* I want information included in the email that an applicant has self-certified as an IGE representative
* so that I know that there is no payment for the request

**Assumptions & Scope**	
What are the assumptions for this story?

Intake currenlty does not know if a request is an IGE request.

Applicants are not filling in the name of the IGE in the Business/Organization field - entering other organization names.

What is IN scope?

Business/Organization field no longer needs to be mandatory
Add a new mandatory field for IGE
Include IGE information in email to Intake

What is NOT in scope?

**Acceptance Criteria**	

**Scenario 1: IGE confirmation statement in email**
* GIVEN an applicant has clicked the IGE checkbox
* WHEN the email is sent to the IAO Intake Inbox
* THEN the email will include this statement " I certify that I am a representative of, and authorized to make a request on behalf of, an Indigenous Governing Entity. An Indigenous Governing Entity is not required to pay application fees."
* AND the email will include the field 'Name of Indigenous Governing Entity'

**Scenario 2: IGE mandatory field**
* GIVEN an applicant is on the contact page
* WHEN the applicant clicks the IGE checkbox
* THEN a mandatory field appears titled 'Name of Indigenous Governing Entity'
* (the existing business/organization field would no longer be mandatory)

**Scenario 3: IGE mandatory field**
* GIVEN an applicant has clicked the IGE checkbox
* WHEN the applicant enters at least one character in the mandatory field appears titled 'Name of Indigenous Governing Entity'
* AND the other mandatory fields have been filled
* THEN the 'Continue' button will turn blue and become active

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by lmullane** [comment]
@jadmsaadaot, @acoard, Intake has asked for additional changes. See revised AC for refinement. cc @abin-aot @m-prodan

**Comment by m-prodan** [comment]
Nice work on this one @jadmsaadaot - QA and regression complete, all credit card scenario and submissions have been tested. Of note, Bambora doesn't have test cards that are expired, but I did use their test cards that include delayed/timed out declined transactions. I also included a test of submitting a payment after sitting idle for 20 minutes and had no issues. Closing this ticket, I'll confirm when we can deploy to PROD
